tests/Functional/FileFromMemoryTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Mailgun\Tests\Functional;
+
+/**
+ * Add attachment with file from memory
+ *
+ * @author Wim Verstuyf <wim.verstuyf@codelicious.be>
+ */
+class FileFromMemoryTest extends \PHPUnit_Framework_TestCase
+{
+    public function testAddFileFromMemory()
+    {
+        $fileValidator = function($files) {
+            $attachments = array_filter($files, function($f) {
```

Comment:
Smart! 
Write a comment here that explains why you are doing this. 

`// Get only the attachments`